test/integration/integration-test.js

```diff
@@ -80,8 +86,16 @@ function teardown(done) {
 describe.skip('integration tests', function() {
   const instructions = {maxLedgerVersionOffset: 100};
   this.timeout(TIMEOUT);
+  const transactions = [];
 
+  // beforeEach(setup);
   beforeEach(setup);
+  beforeEach(function() {
+    if (!this.startLedgerVersion) {
```

Comment:
Ok, I guess it's ok to put in `setup` then. It's probably worth adding a comment to explain why it has to be there.